proto/dream11/od/auth/v1/auth.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package dream11.od.auth.v1;
+
+option go_package = "github.com/dream11/odin/proto/gen/go/dream11/od/auth/v1";
+
+service AuthService {
+  rpc GetUserToken(GetUserTokenRequest) returns (GetUserTokenResponse) {}
+}
+
+message GetUserTokenRequest {
+  string client_id = 1;
+  string client_secret_hash = 2;
```

Comment:
Let's take plain secret also as plain text from user, odin cli will create a hash from it and send it to deployer

Reply:
It is like this only. for the backend it is client_secret_hash (after hashing the secret-access-key set by the user)